src/dbnode/generated/proto/annotation/annotation.proto

```diff
@@ -17,3 +23,17 @@ enum MetricType {
     INFO            = 6;
     STATESET        = 7;
 }
+
+enum SourceFormat {
+    // Using 0 for PROMETHEUS_FORMAT because only Prometheus types were being stored before.
+    PROMETHEUS_FORMAT = 0;
+    UNKNOWN_FORMAT    = 1;
```

Comment:
nit: Perhaps we remove UNKNOWN until we need it? (given we can't make it default value)

Reply:
👍